lib/worker.js

```diff
@@ -164,7 +164,7 @@ Worker.prototype._start = function() {
         self.serviceReturns = res;
         // Make sure that only JSON-serializable values are returned.
         try {
-            return [JSON.parse(JSON.stringify(res))];
+            return JSON.parse(JSON.stringify(res));
```

Comment:
> If you really wanted to match master as it currently is, it should be [JSON.parse(JSON.stringify(res))[0]] but see below.

No, this would only return the startup return from the first service started in the worker, and not all of them. Service-runner supports starting several services in each worker, so we need to return an array of startup returns per worker.